examples/stable_diffusion_xl/gm/modules/embedders/clip/modeling_clip.py

```diff
@@ -334,6 +334,7 @@ class CLIPTextModel(nn.Cell):
     def __init__(self, config_path, weight=None):
         config = self.from_pretrained(config_path)  # CLIPTextConfig
         super().__init__(config)
```

Comment:
Passing `CLIPTextConfig` to `nn.Cell`?

Reply:
I think it's not necessary. I guess when someone translated the torch codes into mindspore codes, the torch `CLIPTextModel` inherited from a parent class `PreTrainedModel`, like [this](https://github.com/huggingface/transformers/blob/main/src/transformers/models/clip/modeling_clip.py#L751). BTW, in `PreTrainedModel`, `self.config` is [initiated](https://github.com/huggingface/transformers/blob/e268d7e5dcfc3a2f070114795592853ab4408473/src/transformers/modeling_utils.py#L1193C9-L1193C29)